src/sourcemaps/tools/wrangler.ts

```diff
@@ -0,0 +1,409 @@
+  options: SourceMapUploadToolConfigurationOptions,
+) {
+  clack.note(
+    chalk.whiteBright(
+      `Configuring source maps upload with Cloudflare Wrangler requires the wizard to:
+- Modify your deploy command to access source maps
+- Set the SENTRY_RELEASE env var to identify source maps
+
+Note: This setup may need additional configuration. 
+We recommend using Vite to build your worker instead, for an easier and more reliable setup.
+
+Learn more about CloudFlare's Vite setup here:
+${chalk.underline(
+  chalk.cyan(
+    'https://developers.cloudflare.com/workers/vite-plugin/get-started/',
+  ),
+)}
+
+You can switch to Vite and re-run this wizard later. 
+Otherwise, let's proceed with the Wrangler setup.`,
+    ),
+    'Before we get started',
+  );
+
+  const proceed = await abortIfCancelled(
+    clack.confirm({
+      message: 'Do you want to proceed with the Wrangler setup?',
+    }),
+  );
+
+  if (!proceed) {
+    await abort(
+      'Got it! You can switch to Vite and re-run this wizard later.',
+      0,
+    );
+    return;
+  }
+
+  await installPackage({
+    packageName: '@sentry/cli',
+    alreadyInstalled: hasPackageInstalled(
+      '@sentry/cli',
+      await getPackageDotJson(),
+    ),
+    devDependency: true,
+  });
+
+  if (!(await askContinueIfHasSentrySourcemapsScript())) {
+    return;
+  }
+
+  const deployCommand = await getDeployCommand();
+  if (!deployCommand) {
+    return;
+  }
+
+  const outDir = await getWranglerOutDir(deployCommand);
+
+  await createAndAddSentrySourcemapsScript({ ...options, outDir });
+
+  await writePostDeployCommand(deployCommand);
+
+  await modifyDeployCommand(deployCommand, outDir);
+
+  await addSentryCliConfig({ authToken: options.authToken });
+}
+
+async function createAndAddSentrySourcemapsScript(
+  options: SourceMapUploadToolConfigurationOptions & { outDir: string },
+) {
+  const pkgJson = await getPackageDotJson();
+  pkgJson.scripts = pkgJson.scripts ?? {};
+  pkgJson.scripts[SENTRY_NPM_SCRIPT_NAME] = getSentryCliCommand(options);
+
+  await fs.promises.writeFile(
+    path.join(process.cwd(), 'package.json'),
+    JSON.stringify(pkgJson, null, 2),
+  );
+
+  clack.log.success(
+    `Added a ${chalk.cyan(SENTRY_NPM_SCRIPT_NAME)} script to your ${chalk.cyan(
+      'package.json',
+    )}.`,
+  );
+}
+
+/**
+ * only exported for testing
+ */
+export function getSentryCliCommand(
+  options: SourceMapUploadToolConfigurationOptions & { outDir: string },
+) {
+  const sentryCliOptions = options.selfHosted ? ` --url ${options.url}` : '';
+  const orgAndProjectArgs = `--org=${options.orgSlug} --project=${options.projectSlug}`;
+
+  return [
+    '_SENTRY_RELEASE=$(sentry-cli releases propose-version)',
+    `sentry-cli${sentryCliOptions} releases new $_SENTRY_RELEASE ${orgAndProjectArgs}`,
+    `sentry-cli${sentryCliOptions} sourcemaps upload ${orgAndProjectArgs} --release=$_SENTRY_RELEASE --strip-prefix '${options.outDir}/..' ${options.outDir}`,
```

Comment:
```suggestion
    `sentry-cli${sentryCliOptions} sourcemaps upload ${orgAndProjectArgs} --release=$_SENTRY_RELEASE --strip-prefix '${options.outDir}${path.delimiter}..' ${options.outDir}`,
```

Reply:
Good catch, thx!

Reply:
ended up extracting this out and using `path.join` which takes care of the right separator and is cleaner.